### PR TITLE
Update Experience alerts docs

### PR DESCRIPTION
**Pull request by gr0**
Hi guys. This is the update to Experience alerts docs. Have a look and see if there is some improvements or adjustments that need to be done before the merge.

In general it looks like this:

preview

### CODE REVIEW

**Comment by otisg** [comment]
I haven't read the above, but I want to comment on the image with annotations.  There will or maybe already are other annotated screenshots.  We don't want to have a mishmash of annotation styles and colours in the docs.  So we need to agree on the style, colour, tool...

CC @adnanrahic

**Comment by gr0** [comment]
Good point @otisg. @adnanrahic, @amir-hadzic let me know if I should remove the annotations.

**Comment by adnanrahic** [comment]
Hey @gr0 , can you use the annotations from [here](https://sematext.com/docs/alerts/creating-logs-alerts/). Or, just don't use any annotations at all.

**Comment by gr0** [comment]
Sure thing, just did the same without annotations:

preview

**Comment by gr0** [comment]
Thanks for the review!

File: docs/alerts/creating-experience-alerts.md [hunk 1]
```diff
@@ -1,11 +1,39 @@
 title: Creating Experience Alerts
 description: Step-by-step alert creation instructions for Experience
 
-Experience App allows you to create alert rules based on [Apdex score](https://en.wikipedia.org/wiki/Apdex) of Page Loads, HTTP Requests and On-Page Transactions.
+Each Experience App metric that you can create alert rules on has a bell icon, when clicked, shows a dropdown menu listing the metrics that you can choose.
```
Comment (otisg) [inline docs/alerts/creating-experience-alerts.md:4]: Each Experience App metric that you can create alert rules on has a bell icon, when clicked, shows a dropdown menu listing the metrics that you can choose. 
==> 
Each Experience App metric that you can create alert rules on has a bell icon that, when clicked, shows a dropdown menu listing the metrics that you can choose.

File: docs/alerts/creating-experience-alerts.md [hunk 2]
```diff
@@ -1,11 +1,39 @@
...
+- Page load backend load time
+- Page load frontend load time
+- Page load time
+- HTTP resource load count
+- HTTP resource load time
+- HTTP resource transfer size
+- User sessions count
+- Users count
+
+The above metrics can also be chosen from in the alert rule creation dialog:
```
Comment (otisg) [inline docs/alerts/creating-experience-alerts.md:28]: The above metrics can also be chosen from in the alert rule creation dialog:
==> The above metrics can also be chosen from the alert rule creation dialog:

File: docs/alerts/creating-experience-alerts.md [hunk 3]
```diff
@@ -1,11 +1,39 @@
...
+
+![Experience Alert Metric](../images/alerts/experience_alert_metrics.png)
+
+While creating an alert you'll get a chart preview of the alert threshold or anomaly. For example, to set up a new alert rule for that would be triggered for page load time higher than 4500 milliseconds on average this is how the preview screen would look like:
```
Comment (otisg) [inline docs/alerts/creating-experience-alerts.md:32]: For example, to set up a new alert rule for that would be triggered for page load time higher than 4500 milliseconds on average this is how the preview screen would look like:

Something's wrong with this sentence...

File: docs/alerts/creating-experience-alerts.md (hunk 1 above)
Comment (gr0) [reply docs/alerts/creating-experience-alerts.md:4]: Adjusted.

File: docs/alerts/creating-experience-alerts.md (hunk 2 above)
Comment (gr0) [reply docs/alerts/creating-experience-alerts.md:28]: Adjusted.

File: docs/alerts/creating-experience-alerts.md (hunk 3 above)
Comment (gr0) [reply docs/alerts/creating-experience-alerts.md:32]: Adjusted.

**Comment by gr0** [comment]
Thanks @otisg. One comment - the whole update went through Grammarly suggestions first. I know it is not perfect, but I try ;)

**Comment by adnanrahic** [comment]
> Spotted a few errors.
> @adnanrahic & @amir-hadzic please read the text if you are reviewers. Or if you approve and didn't read please say that so somebody else can do that or the author can go through the text carefully once more.

My bad, I only skimmed it. Will check more thoroughly next time. FYI, I deployed docs 5min ago, it's up to date with all the latest changes. :slightly_smiling_face:

**Pull request merged**